.gitignore

```diff
@@ -1,6 +1,8 @@
 # Meshery
 mesheryctl/cmd/mesheryctl/mesheryctl
 mesheryctl/.meshery/
+mesheryctl/mesheryctl
```

Comment:
Is this right?

Reply:
It was created when I built mesheryctl using `makefile`

Reply:
Ah, got it. This is the previous built binary before @nitishm introduced a change in structure

Reply:
Fixed!